src/components/Training/Courses.js

```diff
@@ -24,15 +24,19 @@ const Courses = ({ categories, toggleModal }) => {
           <Col width={[1, 1, 1, 1, 1 / 2]} key={cat.id}>
             <Padding bottom={{ smallPhone: 3, tablet: 5 }}>
               <Padding bottom={1}>
-                <Image image={cat.logo} width="60px" />
+                <Image image={cat.logo} width="60px" id={cat.slug} />
                 <SectionTitle>{cat.name}</SectionTitle>
               </Padding>
               <Flex column alignStart>
                 {cat.courses.map(course => (
                   <StyledLink
                     style={{ cursor: 'pointer' }}
-                    onClick={() => toggleModal({ ...cat, ...course })}
+                    to={`/training/${cat.slug}/${course.slug}`}
                     key={course.id}
+                    state={{
```

Comment:
according to the [docs](https://github.com/lsirivong/gatsby-plugin-modal-routing#opening-a-page-in-a-modal)

> gatsby-plugin-modal-routing also provides a `Link` component as a convenience to encapsulate this flag for you.

I think that's a cleaner way (instead of passing the state), what do you think?

Reply:
Yeah it's slightly prettier. I guess I just figured that `asModal` vs `state={{modal: true}}` wasn't a significant enough difference, considering the fact that `StyledLink` already uses gatsby's regular Link component under the hood.

So in order to use `asModal` instead, we would be importing an extra component at the top of this file
`import { Link } from 'gatsby-plugin-modal-routing'` & then passing the `as` prop into `StyledLink`. Or, we could import the modal version of Link directly into our `StyledLink` component & have that one decide which one to use under the hood, depending on whether `asModal` is passed?

Reply:
nevermind, I didn't notice it was a `StyledLink`, that causes more work to be made. I'd keep it as it is then :-)